src/components/Firebase/firebase.js

```diff
@@ -67,9 +72,9 @@ class Firebase {
     this.auth.onAuthStateChanged(authUser => {
       if (authUser) {
         this.user(authUser.uid)
-          .once('value')
+          .get()
           .then(snapshot => {
-            const dbUser = snapshot.val();
+            const dbUser = snapshot.data() || {};
```

Comment:
Normally I think we shouldn't run into the issue of having no db user when a user signs in. Did you run into anything buggy here?

Reply:
Yes, otherwise we have a `Unhandled Rejection (TypeError): Cannot read property 'roles' of undefined`